WebHandler/__init__.py

```diff
@@ -79,7 +80,7 @@ def main_handler(req: func.HttpRequest) -> func.HttpResponse:
     if (req.method.lower() != "post"):
         return func.HttpResponse(
             body=json.dumps(
-                {"status": False, "debugMessage": "Method not supported", "version": version}),
+                {"status": False, "debug_message": "Method not supported", "version": version}),

```

Comment:
json related things make it pascal case everything. 
For the client, we already were using pascal case for everything

Reply:
Everywehre .